develop-docs/backend/api/design.mdx

```diff
@@ -1,57 +1,154 @@
 ---
-title: Designing a New API
+title: Design Principles
 sidebar_order: 2
 ---
 
-[Django REST framework](https://www.django-rest-framework.org/)(DRF) is a powerful and flexible toolkit
-for building Web APIs. Sentry APIs are built using DRF. Here are some considerations to make when designing APIs at Sentry.
+This document contains a set of design principles and requirements whish should be applied to all Sentry APIs. These are _requirements_ when designing public APIs, but internal APIs should attempt to adhere to these principles as well.
```

Comment:
Should this mention 'internal APIs' if all APIs are intended to be public (as per the section starting on line 16).